network-primitives/main.tf

```diff
@@ -0,0 +1 @@
+/Users/vedhavyas/Projects/infra/resources/common.tf
```

Comment:
Is this supposed to be committed?

Reply:
Ahh no. The soft links :(
Will take it out